webshare/share-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with empty ShareData</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        setupManualShareTest(false, '', '', '');
+        callWhenButtonClicked(() => navigator.share({}));
```

Comment:
Could you make this or at least one test verify that value the promise is resolved with? AFAICT this is not actually defined by the spec, probably "resolve p with undefined" is intended.